### ISSUE TITLE
Fix eslint config between react-example and base

### ISSUE THREAD

**Issue by tsamaya** [OpenCageData/geosearch#237]
Since the upgrade of ESLINT (see PR #236), a new conflict is detected running the react example on the eslint configuration
```
ERROR in [eslint] Plugin "react" was conflicted between ".eslintrc.js » plugin:react/recommended" and "BaseConfig » /Users/arnaud/projects/opencage/geosearch/node_modules/react-scripts/node_modules/eslint-config-react-app/base.js".
```

The error message can be dismissed the example would work properly though.

**Comment by tsamaya** [comment]
To prevent the error, run the command at the root of the repo

`yarn upgrade eslint-config-react-app`

This will only patch the local environment and the changes are not stored in the yarn.lock file

**Comment by tsamaya** [comment]
closed by #240
